### ISSUE TITLE
Clarify why demos and demo prep are important

### ISSUE THREAD

**Issue by iteles** [dwyl/process-handbook#60]
I hear this comment today:
> I don’t think [the demo] requires much prep as our client has been following all the way through, demos are more important when the client doesn’t attend the standups

It was for a small project with a fantastic dwyl team and a lovely client, so the impact is not big for _this_ project, but it reminded me that I _clearly_ have done a terrible job communicating the importance of _why_ we do demo prep and demos.

**Comment by iteles** [comment]
Another note to add to this. We have a big client whose product owner is _heavily_ involved in the daily stand ups, knows the backlog like the back of his hand, is in constant contact with the team on individual issues and does a huge proportion of the testing himself. There could not be a more involved client.

Due to a combination of big deadlines, the fact that we're around 35 + sprints into this project, the app is in production under heavy use and some team holidays, there was an experiment around having no demos as the use of this and prep time was questioned.

The result? On Friday I got a call saying that the PO was concerned the team's productivity was decreasing and wasn't sure how we were going to stay ontop of their own client requests.
In fact, the team has been closing issues at the rate of knots and dealing with complex issues quickly and seamlessly. The only thing that has changed? No demos. So the PO has no time at which he stops and takes stock of all that has been achieved in the last 2 weeks.

**Comment by markwilliamfirth** [comment]
@iteles the point that I was making that you quoted me on is that Amanda was following the sprint daily and we were showing her what had been built everyday, so the end of sprint demo was redundant

**Comment by iteles** [comment]
@markwilliamfirth Did you read the comment I made yesterday with a case where exactly that thinking was adopted (with a long standing client, not someone who was fresh to working with us and didn't know the drill) and it caused some unsettling problems?
It was remembering the original thought (which is a good question and I'm sure comes up for everyone who is new to scrum, making it a great addition to this repo) in light of the phone call that urged me to come and update this issue so we can include it as an anecdote in the readme.

**Comment by markwilliamfirth** [comment]
@iteles yes I did read it - what I'm trying to explain is that the situational circumstances are different

**Comment by iteles** [comment]
@markwilliamfirth Please clarify how they were different as from what you said, the difference is that the client who called me is even more involved in the product on a daily basis and would therefore have even less of a reason to be affected by a lack of demos (following your logic as to why they would be "redundant").

**Comment by Cleop** [comment]
@iteles - are there any tangible points that you feel can be added to the readme on this point that aren't already included or can this issue be closed?

**Comment by iteles** [comment]
@Cleop We're doing a great job of the `How?` here: https://github.com/dwyl/process-handbook#sprint-demo but I think we're still very much lacking the `Why?`.
The conversation here is mostly redundant but the main point of this issue still needs to be dealt with.
